server/ai_process.go

```diff
@@ -89,6 +89,7 @@ type aiRequestParams struct {
 	// For live video pipelines
 	segmentReader *media.SwitchableSegmentReader
 	outputRTMPURL string
+	stream        string
```

Comment:
I think we will be adding a bunch more stuff to this struct and part of me wonders if we should split the live pipeline fields into its own sub-struct to keep `aiRequestParams` tidy. Doesn't need to be done in this PR though.

Reply:
Yeah, you can refactor it in the next PR. I think it will be better as you described it.